app/views/coronavirus_landing_page/components/landing_page/_statistic_section_block.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="covid-statistics__statistic-wrapper">
+  <%= render "govuk_publishing_components/components/heading", {
+      text: title,
+      heading_level: 3,
+      font_size: "s",
+      margin_bottom: 1
+    } %>
+
+  <p class="covid-statistics__statistic-value govuk-body">
+    <%= stat %>
+  </p>
+  <p class="govuk-body"><%= subtext %></p>
+
+  <p class="govuk-body">
+    Last updated <%= last_updated_date %>
+  </p>
+
+  <p class="govuk-body">
+    <%= link_to guidance_link[:label], guidance_link[:href], class: "govuk-link" %>
+  </p>
```

Comment:
We should find out if we want any analytics tracking on these (I understand the existing stat links have tracking)

Reply:
As they are external links, I think we track by default.

Reply:
Are you sure? 

Either way, it seems inconsistent to have specific tracking attributes on the ones later in the component: https://github.com/alphagov/collections/pull/2324/files#diff-ba272d011093a246bd1c01ef9a96625abd74ac9c5900afd358f9f058cc3cf791R56-R59

Reply:
Yes so they are tracked by default but happy to add the additional tracking to keep it consistent (I remember the PA saying something about it making it easier for them to track).

Reply:
updated